backend/game_engine/firestore.py

```diff
@@ -280,7 +280,11 @@ def list_challenges(self, challenge_completed_predicate_value=False) -> Iterable
         query = self._client.collection(
             'games/{}/challenges'.format(self._game_id)).where(
                 'complete', '==', challenge_completed_predicate_value)
-        return FirestoreChallengeStream(query.stream())
+        l = []
```

Comment:
logic looks good. please rename l, c and fc for increased readability. may be more difficult to remember what these things do in the future.